packages/react/src/components/field/use-field.ts

```diff
@@ -137,8 +137,9 @@ export const useField = (props: UseFieldProps) => {
       ({
         ...getControlProps(),
         ...parts.input.attrs,
+        'aria-describedby': helperTextId,
```

Comment:
I do believe that `aria-describedby` is already part of the `getControlProps()`, see line 122 of this file.
The issue lies more with the `labelsIds` array not being filled correctly.